File: src/main/node/levels.js
"use strict";

module.exports =
class LevelMap {
  constructor (levels) {
    this.levels = levels.split("SEPARATOR\n");
    this.lastLevel = parseInt(levels,10);
    if (isNaN(this.lastLevel)) {
      this.lastLevel = 1;
    }
  }
  
  getLevel(level) {
    return this.levels[level];
  }
  
  max() {
    return this.lastLevel;
  }
}

File: src/main/python/levels.py
class LevelMaps(object):
    def __init__(self, input_maps):
        self.levels = input_maps.split("SEPARATOR\n")
        try:
            self.max = int(self.levels[0])
        except ValueError:
            self.max = 1

    def max_level(self):
        return self.max

    def get_level(self, level):
        return self.levels[level].rstrip()
